### ISSUE TITLE
A11y Failure: Tab is not tab-able

### ISSUE THREAD

**Issue by rakshith-pibm** [carbon-design-system/carbon-components-react#2380]
When hitting Tab of the Tabs in any browser - it tabs to the next div and not across the tabs.
This is an accessibility violation. 

IBM Cloud Docs: [Issue](https://github.ibm.com/Bluemix/Bluemix-doc-framework/issues/2584)
![Screen Recording 2019-05-23 at 4 29 18 PM](https://user-images.githubusercontent.com/46721470/58288016-14cc6d80-7d78-11e9-8adb-405fd9bca8ca.gif)

Similar behavior on Carbon-component react story book 
![Screen Recording 2019-05-23 at 4 25 20 PM](https://user-images.githubusercontent.com/46721470/58287981-01b99d80-7d78-11e9-9799-5289d2271299.gif)

**Comment by asudoh** [comment]
We had a change there to use arrow key rather than tab key. Hope this makes sense!

**Comment by mekaufman** [comment]
@asudoh Has this been testing with a screen reader? I found when testing this behavior with a screen reader that without the other tabs in the tab order on the page, I didn't even know the other tab menu options existed unless I already knew to use an arrow to move between the options. The screen reader did not tell me 1 of 4 tabs (or menu options) for example, so if I was a blind user, I would have no idea there were other tab options.

**Comment by asudoh** [comment]
Interesting question, given a11y guys tend to bring up W3C practice stuffs in this kind of discussion. That said, what do you think of https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html (from: https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel)?

**Comment by mekaufman** [comment]
@asudoh I think having the selection following focus (auto activation) and https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html options are important based on the needs of the team using the component. 

However, I'm not saying that the use of arrow keys is a problem in and of itself. But, the way it is implemented on our docs homepage, there are some problems. See homepage: https://cloud.ibm.com/docs. I can't tell you if it's implementation or the component. @rakshith-pibm is looking into it further for our implementation. 

But, to help you understand what I'm looking at, let me give you a scenario for the page I just linked above. Maybe you can do a similar test on your component and let me know if arrow keys should go to all options (scenario 1), and what the selector key is for when I use arrow key to get to an option (scenario 2)

Scenario 1
1. Go to the https://cloud.ibm.com/docs page.
2. Using keyboard only, tab to "Get started" tab option.
3. Use arrow keys and try to get to the "Manage" tab option. (this is where things fail for me)

Scenario 2:
1. Go to the https://cloud.ibm.com/docs page.
2. Using the keyboard only, tab to "Get started" tab option.
3. Use right arrow to go to "Build" tab option.
4. Select it using what keyboard key?? **Enter seems to work sporadically, but generally spacebar or Enter do not work for me. Is there a different selector key option?**

**Comment by shixiedesign** [comment]
Hi tagging @dakahn who is familiar with accessibility guidelines on our team. Have you seen this discussion?

**Comment by dakahn** [comment]
Hello! Having tabs be keyboard operable via the arrow keys is not an accessibility violation. See the WAI-aria authoring practices 1.1 tabs example here:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

That said screen reader support as is is wholly inadequate and being addressed by in v10 React in this PR
https://github.com/carbon-design-system/carbon/pull/2852

Those changes should then be backported to v9 versions of the library.
